src/middlewared/middlewared/client/client.py

```diff
@@ -141,7 +141,7 @@ def _unregister_call(self, call):
         self._calls.pop(call.id, None)
 
     def call(self, method, *params, **kwargs):
-        timeout = kwargs.pop('timeout', 30)
+        timeout = kwargs.pop('timeout', 60)
```

Comment:
I saw the timeout only with AD service, especially when we are restarting it! So, maybe a kwargs could be a good idea as far as people don't abuse it.